python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -28,19 +28,21 @@
 import lsst.afw.image as afwImage
 import lsst.meas.algorithms as measAlg
 import lsst.afw.math as afwMath
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
 
-__all__ = ("decorrelateExposure", "computeDecorrelationKernel", "doConvolve",
+__all__ = ("ImageDecorrelationTask", "ImageDecorrelationConfig",
            "computeClippedImageStats")
 
 
-def computeClippedImageStats(im):
+def computeClippedImageStats(im, numSigmaClip=3., numIter=3):
```

Comment:
It is also used by the unit test, which is why I made it public -- but I understand your point. I will do what you suggested, and reimplement it in the unit test.